Intro01/Reflex/src/App.rfx

```diff
@@ -0,0 +1,246 @@
+SERIES_AUTHORITY="datacapture";
```

Comment:
I will, but isn't that better served by the documentation?
https://incapture.atlassian.net/wiki/display/PDT/Rapture+Getting+Started%3A+REFLEX